### DISCUSSION TITLE
Legends of Lightning Tournament Use Cases

### DISCUSSION THREAD

**Discussion by holzeis** [get10101/itchysats#3165] [General]
# Legends of Lightning Tournament

Aiming for the **[Global Adoption Track](https://makers.bolt.fun/tournaments/1/overview)** we need a strong use case that can be implemented in 6 weeks time (with additional time if we reach the finals).

> Focusing on scalability and product market fit, this track allows makers to let their imaginations run wild with either lightning or on-chain technology.

The following list is a summary of our todays discussion.

## tentenone branding
Introduce the new tentenone branding and promote the broader vision. e.g. by extending the wallet functionality, through onboarding etc.

## Integrate with lightning
This could be very simple by supporting a lightning transaction for funding or more sophisticated - see https://github.com/itchysats/meta/issues/78 for more details

## ItchySats goes mobile
Port ItchySats to mobile devices (most likely android) and combine with a lightning or tentenone "branding".

## Watchtower
Prevent e.g. electron users from loosing money in case they can not constantly monitor the chain by running their node.

## Remove online requirement
In comparison to LN markets our biggest disadvantage is the lack of accessibility - also introduced through our always on requirement. This might be more of a feasibility study, but removing that requirement (without compromising our original principles) would be great improvement.

**Comment by klochowicz** [comment]
one more:
allow integration with an existing Lightning Wallet (e.g. Umbrel)
